src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import { useState } from 'react';
 import './App.scss';
+import 'bulma/css/bulma.css';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const getUserById = (userId: number): User | null => {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const usersWithTodo: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+const getNewId = (todos: Todo[]): number => {
```

Comment:
Consider renaming `getNewId` to `getNextTodoId` to better describe what the function does. 🤔